src/components/Spinner.js

```diff
@@ -5,7 +5,7 @@ import spinner from '../common/images/spinner.gif';
 export default function Spinner() {
   return (
     <div className={styles.spinner}>
-      <img src={spinner} width="59" height="54"/>
+      <img src={spinner} width="59" height="54" alt="loading" />
```

Comment:
inline style attributes >:|

Reply:
Seems that this [is recommended](https://stackoverflow.com/a/6784879/763705), browsers use these attributes as "hints for the rendering".

Reply:
took me a while to find that